src/hooks/use-subsocial-api.ts

```diff
@@ -258,6 +267,25 @@ export const useSubSocialApiHook = () => {
     }
   };
 
+  const checkHasTokens = async (account: WalletAccount) => {
+    setLoadingHasToken(true);
+    try {
+      const subsocialApi = await initializeApi();
+      const api = await subsocialApi.substrateApi;
+      if (!api) return null;
+
+      const [balances, energy] = await Promise.all([
+        api.derive.balances.all(account.address),
```

Comment:
Use `system.account` istead of `derive`

And use subcribtion, because we should detect when user take energy after that remove alert and enable button

Reply:
@olehmell If I use system.account, it seems I can't use subscription as it returns promise only
should I just use subscription for the energy one?